setup.py

```diff
@@ -1,29 +1,75 @@
+import sys
+import os
+import codecs
 from setuptools import setup, find_packages
 
-import csp
+
+version = '3.0'
+
+
+if sys.argv[-1] == 'publish':
+    os.system('python setup.py sdist upload')
+    os.system('python setup.py bdist_wheel upload')
+    print('You probably want to also tag the version now:')
+    print('  git tag -a %s -m "version %s"' % (version, version))
+    print('  git push --tags')
+    sys.exit()
+
+
+def read(*parts):
+    filename = os.path.join(os.path.dirname(__file__), *parts)
+    with codecs.open(filename, encoding='utf-8') as fp:
+        return fp.read()
+
+
+install_requires = [
+    'Django>=1.6,<1.10',
+]
+
+
+test_requires = [
+    'pytest==2.9.1',
+    'pytest-django==2.9.1',
+    'pytest-flakes==1.0.1',
+    'pytest-pep8==1.0.6',
+    'pep8==1.4.6',
+    'mock==1.0.1',
+]
+
 
 setup(
     name='django_csp',
-    version=csp.__version__,
+    version=version,
     description='Django Content Security Policy support.',
-    long_description=open('README.rst').read(),
-    author='James Socol',
-    author_email='me@jamessocol.com',
+    long_description=read('README.rst'),
+    author='Christopher Grebs',
```

Comment:
Could maybe use `maintainer` and `maintainer_email` fields instead of updating the author fields?